app/services/keycloak_oauth/get_users_service.rb

```diff
@@ -0,0 +1,42 @@
+require 'net/http'
+
+module KeycloakOauth
+  class GetUsersService < KeycloakOauth::AuthorizableService
+    SUPPORTED_QUERY_PARAMS = %i(briefRepresentation email first firstName lastName max search username)
```

Comment:
Is this the complete list of query params supported by this Keycloak endpoint (e.g. can users have more fields which can be queried here)? If not, it might be hard to debug where the params are stripped. Can we somehow log (warn) the params which are dropped?

Reply:
Yes, it's the complete list (based on https://www.keycloak.org/docs-api/5.0/rest-api/index.html#_userrepresentation). 

Yeah, I think we could add a log for params which are ignored.